contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -27,6 +27,7 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
   error InvalidRequestConfirmations(uint16 have, uint16 min, uint16 max);
   error GasLimitTooBig(uint32 have, uint32 want);
   error NumWordsTooBig(uint32 have, uint32 want);
+  error MsgDataTooBig(uint32 have, uint32 want);
```

Comment:
nit: seems like `max` is a better name than `want` here?  Although the pattern here is using `want`.

`want` seems to suggest that this is the value you should input, while `max` suggests that the value should not exceed this `max`.

Deviating from the pattern might cause issues with off-chain processing later on. Think being consistent here is more important.

Reply:
Done